### PR TITLE
fix: copy changes to frontend

### PR DESCRIPTION
**Pull request by aamyli**
## Brief description. What is this change? 
Copy changes:
- changing all frontend copies to be sentence case
- minor wording changes in some places as well

No code/logic is affected.

## Implementation description. How did you make this change?
* Copy changes

## Steps to test
1. N/A, just make sure I didn't change any code by accident !

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages follow conventional commits and are descriptive. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s) 
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
- [x] The appropriate tests if necessary have been written

### CODE REVIEW

File: frontend/src/components/common/Calendar/WeeklyEventItemPopUp.tsx [hunk 1]
```diff
@@ -118,7 +118,7 @@ const WeeklyEventItemPopUp = ({
                       textStyle={isMobile ? "mobileSmall" : "desktopSmall"}
                       py="0.5rem"
                     >
-                      Next Dropoff:{" "}
+                      Next dropoff:{" "}
```
Comment (aamyli) [inline frontend/src/components/common/Calendar/WeeklyEventItemPopUp.tsx:121]: @Shehryar21 side note idk if this code is still being used but this should be removed right

Comment (Shehryar21) [reply frontend/src/components/common/Calendar/WeeklyEventItemPopUp.tsx:121]: yeah this is not used, will clean up sometime later

File: frontend/src/components/pages/Scheduling/SelectDateTime.tsx [hunk 2]
```diff
@@ -408,7 +408,7 @@ const SelectDateTime = ({
         </>
       )}
       <Text textStyle="mobileHeader2" mt="2em" mb="1em">
-        {isDesktop ? "Drop-off date and time" : "Date and Time"}
+        {"Date and time"}
```
Comment (hanlinc27) [inline frontend/src/components/pages/Scheduling/SelectDateTime.tsx:411]:
```suggestion
Date and time
```

File: frontend/src/components/common/Calendar/WeeklyEventItemPopUp.tsx (hunk 1 above)
Comment (hanlinc27) [reply frontend/src/components/common/Calendar/WeeklyEventItemPopUp.tsx:121]: we're not using this component anymore so removing it or not shouldn't reflect anything for the user ^^

**Review by hanlinc27 — APPROVED** [review]
multitalented! 🤩 small comment

**Review by Shehryar21 — APPROVED** [review]

**Pull request merged**